images/tier1/attendance.svg

```diff
@@ -1,3 +1,6 @@
-<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18" mirror-rtl="true">
-  <path fill="#72777a" d="M17 1H1a1 1 0 0 0-1 1v14a1 1 0 0 0 1 1h16a1 1 0 0 0 1-1V2a1 1 0 0 0-1-1zm-6 4h4a1 1 0 0 1 0 2h-4a1 1 0 0 1 0-2zm0 4h4a1 1 0 0 1 0 2h-4a1 1 0 0 1 0-2zM3 5a2 2 0 0 1 4 0v1a2 2 0 0 1-4 0V5zm0 9.99A.994.994 0 0 1 2 14v-2a3 3 0 0 1 6 0v2a.994.994 0 0 1-1 .99L5 15zm8 .01a1 1 0 0 1 0-2h4a1 1 0 0 1 0 2h-4z"/>
+<svg xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 18 18">
+    <g fill="#72777a" fill-rule="evenodd">
```

Comment:
I can combine them into one path and it seems to render fine then i only have one fill.  I'm curious why you think the <g> is unnecessary, seems like a small compromise between readability and slightly more mark up to group multiple paths.